### PR TITLE
server side log support

### PR DESCRIPTION
**Pull request by sujaykundu777**
added winston with morgan to support server error logs
updated .gitignore to support logs and fonts
update packages
updated readme for error logs
added missing fonts in client (because of which build was failing)

### CODE REVIEW

File: config/winston.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+         ? `${info.timestamp} | [${info.level}] ${info.message} | ${JSON.stringify(info.metadata)}`
+         : `${info.timestamp} | [${info.level}] ${info.message}`)),
+    ),
+    transports: [
+      new transports.File({ filename: './logs/error.log', level: 'error' }),
+    ],
+    exitOnError: false
+  });
+
+export default logger;
```
Comment (nemesisdesign) [inline config/winston.js:20]: new line at the end of the file missing here

File: readme.md [hunk 2]
```diff
@@ -93,6 +93,15 @@ $ npm test 		    	# Run tests
 $ npm test -- -u 		# Update Jest Snapshots
 ```
 
+#### Error Logs :
+
+Server Error logs are generated at /logs/error.log
+
+Run the command to read the logs using Bash (Linux) :
+
+`$ tail -f ./logs/error.log`
```
Comment (nemesisdesign) [inline readme.md:102]: we should simply log to standard output, also, we don't want to log only errors but also info messages and warnings.

**Review by nemesisdesign — CHANGES_REQUESTED** [review]

File: config/winston.js (hunk 1 above)
Comment (sujaykundu777) [reply config/winston.js]: added newLine

File: readme.md (hunk 2 above)
Comment (sujaykundu777) [reply readme.md]: I have added error, warnings and info logs too

File: readme.md [hunk 3]
```diff
@@ -1,143 +1,152 @@
-# openwisp-wifi-login-pages
```
Comment (nemesisdesign) [inline readme.md:1]: can you avoid changing all these lines in the readme please?

File: client/index.css [hunk 4]
```diff
@@ -10,7 +10,7 @@
   font-style: normal;
   font-weight: 600;
   font-display: swap;
-  src: local('Montserrat SemiBold'), local('Montserrat-SemiBold'), url(assets/fonts/montserrat-semi-bold.woff2) format('woff2');
+  src: local('Montserrat SemiBold'), local('Montserrat-SemiBold'), url(assets/fonts/montserrat-semibold-webfont.woff2) format('woff2');
```
Comment (nemesisdesign) [inline client/index.css:13]: avoid this as well please

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
Avoid changes unrelated to the issue you're trying to solve please @sujaykundu777.

File: client/index.css (hunk 4 above)
Comment (sujaykundu777) [reply client/index.css]: Done.reverted back. @nemesisdesign

File: readme.md (hunk 3 above)
Comment (sujaykundu777) [reply readme.md:1]: done reverted back

**Comment by nemesisdesign** [comment]
Sorry, this is too much work for me to correct, I'd rather simply fix the issue on my own.

**Pull request closed without merging**